style.css

```
@@ -0,0 +1,245 @@
+    }
+
+    nav ul #try-now-link {
+      grid-column-start: 6;
+      grid-column-end: 9;
+      text-align: right;
+    }
+
+    nav ul #learn-about-link {
+      grid-column-start: 9;
+      grid-column-end: 11;
+      text-align: center;
+    }
+
+    nav li:hover {
+      color: grey;
+    }
+
+.title-class {
+  background:#F39237;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-content: center;
+  padding: 7em 3em 7em 3em;
+  border: top;
+}
+  .title-class h1 {
+    font-size: 5em;
+  }
+  .title-class sub {
+    padding-bottom: 2em;
+    font-size: 2em;
+  }
+
+  .title-class a {
+    display: block;
+    border: 2px solid black;
+    background-color: lightgrey;
+    font-size: 1em;
+    text-decoration: none;
+    padding: 1em;
+    cursor: pointer;
+    border-radius: 15px;
+    margin: 1em auto;
+    width: 175px;
+    height: 2em;
+  }
+
+.quote-class {
+  background-color: #F2E5D7;
+  padding: 4em 4em;
+  max-height: 15em;
+  font-size: 1em;
+  font-family: fantasy;
+  text-align: center;
+}
+
+  .quote-class blockquote {
+    display: flex;
+    text-align: left;
+  }
+
+  .quote-class p {
+    text-align: right;
+    padding-right: 5em;
+  }
+
+  .quote-class blockquote + p::before {
+    content: "-- "
+  }
+
+.why-class {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+
+  .why-class ul {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+  }
+
+  .why-class li {
+    width: 350px;
+    margin-bottom: 30px;
+  }
+
+  .why-class img {
+    float: left;
+    margin: 10px;
+  }
+
+@media only screen and (max-width: 600px) {
+  .why-class img {
+    display: none;
+  }
+  /* .why-class li strong {
+
+  } */
+}
+  .why-class li strong {
+    margin-top: .5em;
+    float: left;
+  }
+  .why-class li p {
+    text-align: left;
+    padding-bottom: 1em;
+    margin-top: 2.5em;
+  }
+
+.team-class {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+  .team-class ul {
+    display: flex;
+    justify-content: center;
+    flex-flow: row wrap;
+  }
+
+  .team-class span {
+    padding-top: 1em;
+  }
+
+  .team-class li {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 20px 100px 20px 100px;
+  }
+  .team-class img {
+    border-radius: 50%;
+  }
+
+.sponsors-class {
+  align-content: center;
+  padding-bottom: 50px;
+  background: #FFBCB5
+}
+
+  .sponsors-class h2 {
+    margin-block-start: -.83em;
+    border-top: 1px black;
+    padding-top: 100px;
+    font-size: 1.5em;
+  }
+
+  .sponsors-class ul {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    max-width: 1200px;
+    margin: auto;
+  }
+
+  .sponsors-class img {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    margin: 5px;
+  }
+
+.footer-class {
+  background: #3C6E71;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
```

```suggestion
  justify-content: flex-start;
  padding-left: 5em
```
Minor suggestion: you could align the footer links with Startrly (the one at the top) by adding padding-left.